lib/decompress/zstd_decompress.c

```diff
@@ -219,22 +219,23 @@ static size_t ZSTD_frameHeaderSize(const void* src, size_t srcSize)
 }
 
 
-/** ZSTD_getFrameParams() :
+/** ZSTD_getFrameHeader() :
 *   decode Frame Header, or require larger `srcSize`.
-*   @return : 0, `fparamsPtr` is correctly filled,
+*   @return : 0, `fhiPtr` is correctly filled,
```

Comment:
It used to be `ZSTD_frameHeaderInformation`, hence `fhi`.
It was later simplified to `ZSTD_frameHeader`.

I changed the name of this variable, it's now `zfhPtr`.